src/js/common/browserInfoHelper.js

```diff
@@ -41,6 +46,7 @@ export const getDeviceLocalIPAsString = () => {
           reject({message: "NO_IP_FOUND", error: undefined});
         }
         deviceIpString = ip.join(",");
+        deviceIpTimeStamp = new Date().toISOString();
         resolve(deviceIpString);
```

Comment:
I think we need to change how this function resolves, it should now resolve this object:
```js
{
  deviceIpString: 'someValue',
  deviceIpTimeStamp: 'UTC value'
}
```